app/helpers/application_helper.rb

```diff
@@ -302,8 +302,8 @@ def edit_inline(object, property, options = {})
     editable(object, property, {:type => type, :title => title, :value => value, :class => klass, :source => select_values, :url => update_url, :placeholder => placeholder}.compact)
   end
 
-  def documentation_url(section = "", options = {})
-    main_app.external_link_url(options.merge(type: 'manual', params: { section: section }))
+  def documentation_url(section = nil, type: 'manual', **options)
```

Comment:
I think this should be safe, but I'm not 100% sure.

Reply:
Works fine!